Tests/SonarQube.TeamBuild.PreProcessor.Tests/ArgumentProcessorTests.cs

```diff
@@ -95,14 +93,16 @@ public void PreArgProc_PropertiesFileSpecifiedOnCommandLine()
         {
             // 0. Setup
             string testDir = TestUtils.CreateTestSpecificFolder(this.TestContext);
-            string propertiesFilePath = Path.Combine(testDir, ActualRunnerPropertiesFileName);
+            string propertiesFilePath = Path.Combine(testDir, "mysettings.txt");
```

Comment:
The changes in this file are minor test cleanup. The test had been changed to test the new /s: switch, but bits of the test still refer to the sonar-runner.properties file.